### ISSUE TITLE
Replace Mocks with Fakes in tests

### ISSUE THREAD

**Issue by victornoel** [zold-io/java-api#42]
There is a lot of use of Mockito in the tests, it makes it very hard to read and a pain to write.

Using Fakes (see https://www.yegor256.com/2014/09/23/built-in-fake-objects.html) instead would improve the quality of the tests as well as provide a secondary implementation of the main concepts of the Zold java-api, which can only be good for the project.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/CAZUREFND) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/CAZUREFND) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zold-io/java-api/issues/42#issuecomment-406785267))

@g4s8 This is what I know about this job in [CAZUREFND](https://www.0crat.com/p/CAZUREFND), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#42](https://github.com/zold-io/java-api/issues/42) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/CAZUREFND?q=%7Bjob%3A%27gh%3Azold-io%2Fjava-api%2342%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by carlosmiranda** [comment]
@victornoel will implement this after #49, we can use `Score.Default` from there to replace some mocks.

**Comment by carlosmiranda** [comment]
@0crat wait for #49, that PR has some useful code we can use here.

**Comment by 0crat** [comment]
> @0crat wait for #49, that PR has some useful code we can use here. ([here](https://github.com/zold-io/java-api/issues/42#issuecomment-408303221))

@carlosmiranda The impediment for [#42](https://github.com/zold-io/java-api/issues/42) was registered successfully by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by carlosmiranda** [comment]
@victornoel see PR #64 please

**Comment by carlosmiranda** [comment]
@victornoel you are better off commenting this (https://github.com/zold-io/java-api/issues/42#issuecomment-411422843) in #64 than here. :)

**Comment by carlosmiranda** [comment]
@0crat refuse

**Comment by carlosmiranda** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zold-io/java-api/issues/42#issuecomment-540871690))

@carlosmiranda The user @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) resigned from [#42](https://github.com/zold-io/java-api/issues/42), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zold-io/java-api/issues/42#issuecomment-540871690))

@carlosmiranda There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@9b7f0750-f619-44d8-bdbc-fcfc0dc4b9ec, thread: pool-3-thread-2
com.zerocracy.Policy[105] java.lang.IllegalArgumentException: Policy item '6.penalty' not found
```

[0.49.18](https://github.com/zerocracy/farm): CID: [f41d22b4-8c0e-4a17-a22a-745ff913f2fc](https://www.0crat.com/footprint/CAZUREFND/f41d22b4-8c0e-4a17-a22a-745ff913f2fc), Type: "Order was canceled"
